ZenPacks/zenoss/Microsoft/Windows/datasources/PerfmonDataSource.py

```diff
@@ -756,6 +758,12 @@ def onReceiveFail(self, failure):
             self.receive()
             defer.returnValue(None)
 
+        elif "Attempting to perform the InitializeDefaultDrives operation on the 'FileSystem' provider failed." in e.message:
+            retry, level, msg = (
+                True,
+                logging.WARN,
```

Comment:
let's change the logging to DEBUG.  and also the message.  "failure" in the message will alarm people and cause support to waste time.  how about "Ignoring powershell error on {} as it does not affect collection: {}"

Reply:
great, sure